pages/hackathon/[slug]/index.tsx

```diff
@@ -107,7 +119,9 @@ export default function Hackathon() {
         )
     }
 
-    var myTeam = { team_id: "#" }
+    var myTeam: TeamSerializer | TeamSerializer[] | { team_id: string } = {
+        team_id: "#",
+    }
```

Comment:
No need for array, for now, I guess :thinking:. Currently, only one team per user per hackathon is allowed.

Reply:
Since we are only using this for getting the link of the team, we can instead change this variable to `string` and store only the `team_id` (used in the link of the team). 

Which of these do you think will be better - current or this :thinking:?

Reply:
This approach will be better ig. I will modify this

Reply:
done